styleguide.config.js

```diff
@@ -65,6 +71,9 @@ module.exports = {
         },
         {
           src: "//baltimorecountymd.gov/sebin/r/z/dotgov-steplist.min.js"
+        },
+        {
+          src: "http://baltimorecountymd.gov/sebin/d/n/dotgov-modal.min.js"
```

Comment:
Referring to User Story 14770 > acceptance criteria > Scenario: User closes a modal; the user should be able to close the modal by clicking outside/ in the gray area around the modal to close as well. Recommend adding a toggle functionality for `.dialog-backdrop` to the dotgov-modal.min.js that mimics the functionality targeting the id `my-accessible-dialog-close`. If we want to pursue something like this, recommend moving `.dialog-backdrop` from a class to an id for organization, since it will probably need to be targeted by the javascript function.

Reply:
@theilman7 can you check if this is an either or, or the user should always be able to click outside the gray. I have never been a fan of clicking outside in the case of a modal, because you can accidentally close it.

Reply:
@martypowell @theilman this seems like something that would be on a case by case basis. Any modal/popup that requires a user to fill out information or click action buttons to continue shouldn't allow a background click to close as a user can lose information or become lost easily. Any informational modal's such as instructions or images I could see having a background click to close as its kind of expected and you don't run the risk of losing data. Both should be able to use ESC or Enter to close along with an X button in the corner. That's how I see it anyway.

Reply:
@tmccoy529 i think you are mostly right, and i believe this is how most modals are configured. My question to you and those other libraries are wouldn't that be confusing for the user?

Reply:
@baltimorecounty/front-end-developers probably needs to go to design team.

Reply:
Referring to the link sent earlier, I see thats how W3 has theres setup as well so that should be fine. In the [user story](https://oittfs.bcg.ad.bcgov.us/BAUCollection/BaltimoreCounty.Gov%20Website/_workitems/edit/14770) it states the following as one of the acceptance criteria, scenario based examples: Scenario: User closes a modal.
Given: A modal is active.
When: The user selects outside the modal or selects the close button.
Then: The modal disappears. Since the criteria states "The user selects outside the modal or selects the close button", this line should be updated on the acceptance criteria for when testing time comes.